packages/react-components/tests/browser/VideoGallery.spec.tsx

```diff
@@ -6,12 +6,9 @@ import { test, expect } from '@playwright/experimental-ct-react';
 import { VideoGallery } from '../../src/components/VideoGallery';
 import { VideoGalleryLocalParticipant, VideoGalleryRemoteParticipant } from '../../src';
 import { Stack } from '@fluentui/react';
+import { Locator } from 'playwright-core';
 
 test.describe('VGL - VideoGallery tests', () => {
-  test.beforeEach(async ({ page }) => {
-    await page.evaluate(() => document.fonts.ready);
-  });
```

Comment:
Why are we removing this beforeEach?

Reply:
We should do this after the mount, to ensure that the fonts used by the component are fully loaded.
I've moved this to tests instead